README.md

```diff
@@ -100,3 +100,83 @@ make all-tests
 ### Environment Variables
 
 Placeholder
+
+# Survey Assist SIC LLM - Evaluation Methodology
```

Comment:
Given that you primarily focus on alignment between CC and SA outputs, make the title more informative (e.g. Methodology for evaluating alignment between clerical coders and Survey Assist outputs)